third_party/terraform/website/docs/provider_reference.html.markdown

```diff
@@ -278,3 +287,22 @@ as their versioned counterpart but that won't necessarily always be the case.
 [service accounts]: https://cloud.google.com/docs/authentication/getting-started
 [GCE metadata]: https://cloud.google.com/docs/authentication/production#obtaining_credentials_on_compute_engine_kubernetes_engine_app_engine_flexible_environment_and_cloud_functions
 [scopes]: https://developers.google.com/identity/protocols/googlescopes
+
+---
+
+* `batching` - (Optional) Controls batching for specific GCP request types where users have encountered quota or speed issues using `count` with resources that affect the same GCP resource (e.g. `google_project_service`). It is not used for every resource/request type and can only group parallel similar calls for nodes at a similar traversal time in the graph during `terraform apply` (e.g. resources created using `count` that affect a single `project`). Thus, it is also bounded by the `terraform` [`-parallelism`](https://www.terraform.io/docs/commands/apply.html#parallelism-n) flag, as reducing the number of parallel calls will reduce the number of simultaneous requests being added to a batcher.
+
+~> **NOTE**: Not every resource or request has had batching implemented for it.
```

Comment:
I don't know exactly what resources we are going to support in the future. We certainly can be explicit in the provider docs, I just didn't know if it would make sense later.

Reply:
The way it's written makes it sound like many resources are supported when I expect that only a small handful will be. You could add a link here that would expand the list over time. eg: Add the batching support to each resource's markdown docs and add a link here like https://github.com/terraform-providers/terraform-provider-google/search?q=batching+extension%3Amarkdown&unscoped_q=batching_support